modules/glossary-of-common-terms-for-rhoai.adoc

```diff
@@ -0,0 +1,85 @@
+// Reference: RHEL AI glossary, https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x9094307
+
+graphics processing unit (GPU):: A specialized processor designed to rapidly manipulate and alter memory to accelerate the creation of images in a frame buffer intended for output to a display. GPUs are heavily utilized in machine learning due to their parallel processing capabilities.
+//Reference IBM Glossary https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x8987320
+
+inference:: The process of using a trained AI model to generate predictions or conclusions based on the input data provided to the model. 
+
+inference server:: A server that helps an AI model make new conclusions based on its prior training. Inference servers feed the input requests through a machine learning model and return an output.
+
+large language model (LLM):: A language model with a large number of parameters, trained on a large quantity of text.
+//Reference: https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x10298052
+
+machine learning (ML):: A branch of artificial intelligence (AI) and computer science that focuses on the use of data and algorithms to imitate the way that humans learn, gradually improving the accuracy of AI models.
+
+model:: In a machine learning context, a set of functions and algorithms that have been trained and tested on a data set to provide predictions or decisions. 
+// https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x2245601
+
+model registry:: A central repository containing metadata related to machine learning models from inception to deployment. The metadata ranges from high-level information like the deployment environment and project origins, to intricate details like training hyperparameters, performance metrics, and deployment events.
+// Reference: https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x8397498
+
+MLOps:: The practice for collaboration between data scientists and operations professionals to help manage production machine learning (or deep learning) lifecycle. MLOps looks to increase automation and improve the quality of production ML while also focusing on business and regulatory requirements. It involves model development, training, validation, deployment, monitoring, and management and uses methods like CI/CD.
+//Reference: https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x10072886
+
+notebook:: An interactive document that contains executable code, descriptive text for that code, and the results of any code that is run.
+// Reference https://dataplatform.cloud.ibm.com/docs/content/wsj/wscommon/glossary-wx.html?context=wx#x2031718
```

Comment:
The full name is Notebook interface or Computational notebook https://en.wikipedia.org/wiki/Notebook_interface